configure.ac

```diff
@@ -49,7 +49,7 @@ PKG_CHECK_MODULES(INITIAL_SETUP,
                   pango >= $PANGO_REQUIRED_VERSION
                   rest-0.7
                   json-glib-1.0
-		  libsecret-1
+                  libsecret-1
```

Comment:
Since these are obviously correct, I've pushed them upstream. I also checked to see if any more unnecessary newlines were introduced since gnome-3-22 (there were not).